astro/src/content/quickstarts/quickstart-dotnet-web.mdx

```diff
@@ -0,0 +1,325 @@
+
+<Aside type="note">
+If you simply want to run the application , there is a completed version in the 'complete-application' directory.  You can use the following commands to get it up and running if you do not want to create your own.
+
+```shell
+cd complete-application
+dotnet publish -r osx-x64
+dotnet run
+```
+
+Then view the website at the following location `http://localhost:5000`.
+</Aside>
+
+Since the complete application already exists in the `complete-application` directory, you will need to delete all files in the `compete-application` directory or choose a directory other than `fusionauth-quickstart-dotenet-web` to build the new application. 
+
+Next open your command-line shell (i.e. Terminal on a Mac or Powershell on Windows) and navigate to the directory of your choice.  Then create a new web application using the `dotnet` CLI and go to that directory:
+
+```shell
+dotnet new webapp -o complete-application
+cd complete-application
+```
+
+<Aside type="note">
+If you change the name of the application from 'complete-application' to some other name, you will need to make sure the correct namespace is used in all of your code.
+</Aside>
+
+Creating a new application in Visual Studio will work as well.  You will use the command line here for the simplicity of the example.
+
+To see the results, you should publish this application and run it. There are [multiple ways of deploying an application](https://docs.microsoft.com/en-us/dotnet/core/deploying/), but publishing ensures your deployment process is repeatable. You can use the [RID catalog](https://learn.microsoft.com/en-us/dotnet/core/rid-catalog) to build different versions of this application for different platforms. Here's the command to publish a standalone executable you could deploy behind a proxy like Nginx:
+
+```shell
+dotnet publish -r osx-x64
+```
+
+Then start up the web application.
+
+```shell
+dotnet run
+```
+![Shell Showing Hosting Port](/img/articles/quickstart-dotnet-web/shell-window-showing-host-port.png)
+
+The application will be accessible at `http://localhost:{PortNumber}` where the port number will be displayed in your shell window.  I.E. `http://localhost:5178`
+
+![Base Application Home Page](/img/articles/quickstart-dotnet-web/dotnet-new-app-welcome-page.png)
+
+## Authentication
+
+It's always smart to leverage existing libraries as they are likely to be more secure and better handle edge cases. You're going to add an `OpenIdConnect` library to the application. Make sure you're in the `complete-application` directory and run the command below to add the library.
```

Comment:
Should I start a new terminal window? should I control-c the `dotnet run` command?

Reply:
I can see the confusion.  I added a step to explain that control-c will stop the hosting above and that should put them in a good state.